src/libponyrt/lang/stat.c

```diff
@@ -69,6 +71,8 @@ static void windows_stat(const char* path, pony_stat_t* p, struct __stat64* st,
   p->broken = false;
 
   p->hard_links = (uint32_t)st->st_nlink;
+  p->device = (uint32_t)st->st_dev;
```

Comment:
negative device number? thats insane!  It should not be signed for sure, maybe `uint64_t` ?